diff --git a/node_modules/sapper/dist/chunk6.js b/node_modules/sapper/dist/chunk6.js
index 6f32731..f5f6738 100644
--- a/node_modules/sapper/dist/chunk6.js
+++ b/node_modules/sapper/dist/chunk6.js
@@ -4,22 +4,22 @@ var htmlMinifier = require('html-minifier');
 
 function minify_html(html) {
 	return htmlMinifier.minify(html, {
-		collapseBooleanAttributes: true,
-		collapseWhitespace: true,
-		conservativeCollapse: true,
-		decodeEntities: true,
-		html5: true,
-		ignoreCustomComments: [/^#/],
-		minifyCSS: true,
-		minifyJS: false,
-		removeAttributeQuotes: true,
-		removeComments: true,
-		removeOptionalTags: true,
-		removeRedundantAttributes: true,
-		removeScriptTypeAttributes: true,
-		removeStyleLinkTypeAttributes: true,
-		sortAttributes: true,
-		sortClassName: true
+		// collapseBooleanAttributes: true,
+		// collapseWhitespace: true,
+		// conservativeCollapse: true,
+		// decodeEntities: true,
+		// html5: true,
+		// ignoreCustomComments: [/^#/],
+		// minifyCSS: true,
+		// minifyJS: false,
+		// removeAttributeQuotes: true,
+		// removeComments: true,
+		// removeOptionalTags: true,
+		// removeRedundantAttributes: true,
+		// removeScriptTypeAttributes: true,
+		// removeStyleLinkTypeAttributes: true,
+		// sortAttributes: true,
+		// sortClassName: true
 	});
 }
 
